security/src/main/java/io/micronaut/security/authentication/DefaultAuthorizationExceptionHandler.java

```diff
@@ -86,16 +85,15 @@ public DefaultAuthorizationExceptionHandler(RedirectConfiguration redirectConfig
                 return HttpResponse.serverError();
             }
         } else {
-            return httpResponseWithStatus(request, exception);
+            return httpResponseWithStatus(exception);
         }
     }
 
     /**
-     * @param request The request
      * @param exception The exception
      * @return The response to be used when a redirect is not appropriate
      */
-    protected MutableHttpResponse<?> httpResponseWithStatus(HttpRequest request, AuthorizationException exception) {
+    protected MutableHttpResponse<?> httpResponseWithStatus(AuthorizationException exception) {
```

Comment:
This is breaking change. Moreover, the request is being passed intentionally to allow people override `httpResponseWithStatus` to have access to it.

Reply:
I mistakenly removed it. Good catch!